azure-proto-core/SubscriptionContainerOperations.cs

```diff
@@ -0,0 +1,46 @@
+using Azure;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using azure_proto_core.Adapters;
+using System;
+using System.Threading;
+
+namespace azure_proto_core
+{
+    /// <summary>
+    /// Subscription Container Operations
+    /// </summary>
+    public class SubscriptionContainerOperations : OperationsBase
+    {
+        public static readonly string AzureResourceType = "Microsoft.Resources/subscriptions";
+
+        public SubscriptionContainerOperations(ArmClientContext context, string defaultSubscription) : base(context, $"/subscriptions/{defaultSubscription}") { }
+
+        public SubscriptionContainerOperations(ArmClientContext context, ResourceIdentifier id) : base(context, id) { }
+
+        public SubscriptionContainerOperations(ArmClientContext context, Resource subscription) : base(context, subscription) { }
+
+        public override ResourceType ResourceType => AzureResourceType;
+
+        public Pageable<SubscriptionOperations> List(CancellationToken cancellationToken = default)
```

Comment:
Didn't these exist as methods in armclient?  Should they be removed since list now lives here?

Reply:
Yes they should be removed